### DISCUSSION TITLE
测试完成，未见缓存文件增大

### DISCUSSION THREAD

**Discussion by jaesony** [kkkgo/PaoPao-Pref#4] [测试分享]
root@Debian:~# docker run --rm --net=host -it sliamb/paopao-pref -server 192.168.0.253 -pc 1 -limit 25 -sleep 0ms -timeout 1s
Percentage : 1 %
Total Line : 8480
Concurrency Limit: 25
Timeout: 1s
Sleep: 0s
DNS Server: 192.168.0.253
DNS Port: 53
Nslookup 8518/8480 domains[100.4481%]. Succ rate:99.44%. Avg time: 0.002668 seconds. Est time: 00:00:00.
Done. Processed 8518/8480 domains in 23.73 seconds.
看了下paopaodns的缓存文件redis_dns_v2.rdb大小保持不变，也没生成redis_dns.rdb文件

**Comment by kkkgo** [comment]
Percentage : 1 %，数据太小。

**Comment by jaesony** [comment]
现在redis_dns_v2.rdb增加了25M左右，1%的域名。好像是延迟原因，延迟怎么有近10分钟？

**Reply by kkkgo** [reply]
https://github.com/kkkgo/PaoPaoDNS/discussions/12#discussioncomment-5749160

**Reply by jaesony** [reply]
OK
